be/src/formats/orc/orc_chunk_reader.cpp

```diff
@@ -1842,7 +1842,7 @@ void OrcChunkReader::_add_conjunct(const Expr* conjunct, std::unique_ptr<orc::Se
 #define ADD_RF_INT_TYPE(type)                                          \
     case type: {                                                       \
         auto* xrf = dynamic_cast<const RuntimeBloomFilter<type>*>(rf); \
-        if (xrf == nullptr) return false;                              \
+        if (xrf == nullptr || !xrf->has_min_max()) return false;       \
```

Comment:
`!xrf->has_min_max()`   this check is strange.

Reply:
Why? If this runtime filter is empty, possibliy there is no min/max value. It's a defensive programming action.